comitup/nm.py

```diff
@@ -89,8 +90,11 @@ def device_name(device):
 
 def get_wired_device():
     dev = [x for x in get_devices() if x.DeviceType == 1]
-    if dev: return dev[0]
-    return None
+
+    if not dev or not getattr(conf, 'manage_wired_device', False):
```

Comment:
I'd like to see the key initialized in the original config invocation, set to false